avae/model_b.py

```diff
@@ -317,10 +317,7 @@ def forward(self, x):
         # reparametrise
         latent = self.sample(latent_mu, latent_logvar)
         # decode
-        if self.pose:
-            x_recon = self.decoder(latent, latent_pose)
-        else:
```

Comment:
does this change need to be added to model_a as well?

Reply:
Yes, possibly! I haven't ran these changes on model A at all.

Reply:
No, this only removes some redundancy (note ```latent_pose``` variable has already been initialised to None on [line 316](https://github.com/alan-turing-institute/affinity-vae/blob/51-fix-betaVAE-bug/avae/model_b.py#L316).